hax/hax/handler.py

```diff
@@ -50,10 +51,31 @@ def __init__(self, q: Queue, motr: Motr):
         self.is_stopped = False
         self.consul = ConsulUtil()
         self.eq_publisher = EQPublisher()
+        self.statuses: List = []
```

Comment:
It seems like this `statuses` is obsolete and not actually used. Is it correct?

Reply:
Removed.